docs/documentation/platform/dynamic-secrets/mssql.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: "MS SQL"
+description: "How to dynamically generate MS SQL database users."
+---
+
+The Infisical MS SQL dynamic secret allows you to generate Microsoft SQL server database credentials on demand based on configured role.
+
+## Prerequisite
+
+Create a user with the required permission in your SQL instance. This user will be used to create new accounts on-demand.
+
+
+## Set up Dynamic Secrets with MS SQL
+
+<Steps>
+  <Step title="Open Secret Overview Dashboard">
+	Open the Secret Overview dashboard and select the environment in which you would like to add a dynamic secret.
+  </Step>
+  <Step title="Click on the 'Add Dynamic Secret' button">
+	![Add Dynamic Secret Button](../../../images/platform/dynamic-secrets/add-dynamic-secret-button.png)
+  </Step>
+  <Step title="Select `SQL Database`">
+	![Dynamic Secret Modal](../../../images/platform/dynamic-secrets/dynamic-secret-modal.png)
```

Comment:
let's update the image used here to latest? 
<img width="1263" alt="image" src="https://github.com/Infisical/infisical/assets/65645666/da94a308-327e-4461-a198-623d9570769e">